test/mixin/TransientStorageArray.t.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8;
+
+import {Test, console} from "forge-std/Test.sol";
+
+import {BusyStoreAndRead, BusyTransientStorage} from "./TransientStorageArray/BusyTransientStorage.sol";
+import {
+    BYTES_IN_WORD,
+    ExposedTransientStorageArray,
+    StoreAndRead,
+    sequentialByteArrayOfSize
+} from "./TransientStorageArray/ExposedTransientStorageArray.sol";
+
+contract TransientStorageArrayTest is Test {
+    uint256 seed = 0;
+
+    StoreAndRead executor;
+
+    function pseudorandomByteArrayOfSize(uint256 length) internal returns (bytes memory data) {
+        uint256 currentSeed = seed;
+        data = new bytes(length);
+        for (uint256 i = 0; i < length; i++) {
+            data[i] = abi.encode(keccak256(abi.encodePacked(currentSeed, length, i)))[0];
+        }
+        seed = currentSeed + 1;
+    }
+
+    function setUp() public {
+        executor = new StoreAndRead(new ExposedTransientStorageArray());
+    }
+
+    function checkRoundTrip(bytes memory data) private {
+        assertEq(executor.storeAndRead(data), data);
+    }
+
+    function test_emptyArray() external {
+        checkRoundTrip(hex"");
+    }
+
+    function test_shortArray() external {
+        checkRoundTrip(hex"1337");
+    }
+
+    function test_singleWordArray() external {
+        checkRoundTrip(sequentialByteArrayOfSize(BYTES_IN_WORD));
+    }
+
+    function test_oneByteAndASingleWordArray() external {
+        checkRoundTrip(sequentialByteArrayOfSize(BYTES_IN_WORD + 1));
+    }
+
+    function test_largeArrayWordMultiple() external {
+        checkRoundTrip(sequentialByteArrayOfSize(BYTES_IN_WORD * 1337));
+    }
+
+    function test_largeArrayNotWordMultiple() external {
+        checkRoundTrip(sequentialByteArrayOfSize(BYTES_IN_WORD * 1337 + 1));
```

Comment:
I didn't get the difference between this test and the previous one. What's the deal with `1337` number?